packages/utils/src/fhir-converter/convert.ts

```diff
@@ -154,8 +154,6 @@ function replaceIDs(fhirBundle: FHIRBundle, patientId: string): FHIRBundle {
     const regex = new RegExp(stringToReplace.old, "g");
     fhirBundleStr = fhirBundleStr.replace(regex, stringToReplace.new);
   }
-
-  console.log(`Bundle being sent to FHIR server: ${fhirBundleStr}`);
```

Comment:
Intentional? Wasn't it working, or weren't we using it?

Reply:
Intentional. I was running the FHIR-converter e2e test, and imagine getting full bundles logged for 10k files..